templates/workloads/partials/cf/envvars.yml

```diff
@@ -23,6 +23,17 @@ Environment:
 - Name: COPILOT_SNS_TOPIC_ARNS
   Value: '{{jsonSNSTopics .Publish.Topics}}'
 {{- end}}{{- end}}
+{{- if .Subscribe}}{{- if .Subscribe.Topics}}
+- Name: COPILOT_QUEUE_URIS
+  Value: !Sub
+    - '{{jsonQueueURIs .Subscribe.Topics}}'
+    - mainURL: !Ref EventsQueue
+    {{- range $topic := .Subscribe.Topics}}
+    {{- if $topic.Queue}}
+      {{$topic.Service}}-{{$topic.Name}}URL: !Ref {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue
```

Comment:
Does it render properly when `$topic.Service` or `$topic.Name` is `nil`? I don't recall 😛

Reply:
If nil it should skip them as they need a valid name and service to be subscribed to. I added more ifs.